### PR TITLE
webui: storage step improvements

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: ui/webui/src/actions/storage-actions.js [hunk 1]
```diff
@@ -32,36 +32,31 @@ import {
 export const getDevicesAction = () => {
     return async (dispatch) => {
         const devices = await getDevices();
-        return Promise.all(devices[0].map(device => dispatch(getDeviceDataAction({ device }))));
-    };
-};
+        const devicesData = await Promise.all(devices[0]
+                .map(device => device)
```
Comment (rvykydal) [inline ui/webui/src/actions/storage-actions.js:36]: This step is puzzling me.

Comment (KKoukiou) [reply ui/webui/src/actions/storage-actions.js:36]: Oops that's wrong. Sorry.

Comment (KKoukiou) [reply ui/webui/src/actions/storage-actions.js]: Fixed.

**Review by rvykydal — APPROVED** [review]
Looks good to me, thank you! Nice cleanup. Must have been fun to hunt the flakes :)

**Review by adamkankovsky — APPROVED** [review]
Looks fine to me. I'll probably have a lot of conflicts with wizard in another PR, but it makes more sense this way.

**Review by VladimirSlavik — APPROVED** [review]
LGTM, thank you!

**Pull request merged**